monitor/grafana/zeebe-overview.json

```diff
@@ -1,4 +1,52 @@
 {
+  "__inputs": [
+    {
```

Comment:
:thought_balloon: Wondering why this changed (or was added) where and how did you export the dashboard?

Reply:
Yeah I used the local testing guide this time. I'll open an issue to align these, because IMO the local testing guide should be a reasonable starting point as well.